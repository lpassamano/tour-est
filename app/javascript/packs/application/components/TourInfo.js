import React, { Component } from "react";

class TourInfo extends Component {
  render() {
    return (
      <div>
        <h4>{this.props.title}</h4>
      </div>
    );
  }
}

//TODO: add proptypes
// add back into tourlist component

export default TourInfo;
