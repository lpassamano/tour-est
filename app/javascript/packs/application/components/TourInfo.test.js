import TourInfo from "./TourInfo";
import React from "react";
import { mount } from "enzyme";

describe("<TourInfo />", () => {
  test("render", () => {
    const component = mount(<TourInfo />);
    expect(component).toMatchSnapshot();
  });
});
