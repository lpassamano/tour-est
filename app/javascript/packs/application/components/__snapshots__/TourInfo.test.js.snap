// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TourInfo /> render 1`] = `
<TourInfo>
  <div>
    <h4 />
  </div>
</TourInfo>
`;
